src/ProjectTemplates/Web.ItemTemplates/content/RazorPage/.template.config/template.json

```diff
@@ -37,11 +37,13 @@
   "symbols": {
     "namespace": {
       "description": "namespace for the generated code",
+      "displayName": "Namespace",
       "replaces": "MyApp.Namespace",
       "type": "parameter"
     },
     "no-pagemodel": {
       "description": "create page without a PageModel",
+      "displayName": "Exclude PageModel",
```

Comment:
This won't strictly show up anywhere right now since we don't support parameters in item templates, but this is a good change